Packs/CommonScripts/ReleaseNotes/1_12_30.md

```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### TextFromHTML
+
+- Added the *allow_body_fallback* argument, which allows to use the full *html* as the body in case the input *html* does not have a `body` tag
+- Added the *replace_line_breaks* argument, which allows to replace `br` tags in the html with line breaks in the extracted text
+- Added the *trim_result* argument, which allows to replace leading and trailing whitespaces as well as collapsing multible empty lines in the extracted text to a single empty line
+- Added the *output_to_context* argument, which allows to store the extracted text in context
```

Comment:
```suggestion
- Added the *allow_body_fallback* argument, which allows using the full *html* as the body in case the input *html* does not have a `body` tag.
- Added the *replace_line_breaks* argument, which allows replacing `br` tags in the html with line breaks in the extracted text.
- Added the *trim_result* argument, which allows replacing leading and trailing whitespaces as well as collapsing multiple empty lines in the extracted text to a single empty line.
- Added the *output_to_context* argument, which allows storing the extracted text in context.
```